fuseki/sample-ubd.trig

```diff
@@ -386,8 +386,7 @@ graph <cube-project/ubd/csv-mapping/table-station> {
     cc:csvw <cube-project/ubd/csv-mapping/table-station/csvw> ;
     schema:name "Station" ;
     schema:color "#BBBBBB" ;
-    # cc:identifierTemplate "station/{STATION_ID}" ;
-    cc:identifierTemplate "{station_id}-{year}-{aggregation_id}" ;
+    cc:identifierTemplate "station/{STATION_ID}" ;
```

Comment:
The `STATION_ID` variable is wrong now that you downcased the column names

Reply:
Done. Although we might want to sync the test dataset and the sample file